src/upgrade.jl

```diff
@@ -124,6 +124,7 @@ function _upgrade_env(pkgs::Dict{String, VersionNumber})
     try
         mktempdir() do tmp
             Pkg.activate(tmp)
+            isempty(pkgs) && return _env()
```

Comment:
why is this change here?

Reply:
Necessary to get tests to pass on julia 1.5 / nightly. I can move it to a separate pull request though.

Reply:
i have no idea what this is doing.

Reply:
The `require_not_empty` was added to Pkg.jl for 1.5. This creates an early exit condition to work around that.

Reply:
https://github.com/JuliaLang/Pkg.jl/commit/87aab4a2b5e1e9213e4cde9f8049420a8e0e647f#diff-a43f827629f8ae12bbcbb218a293d325afcdda3a7f3c7016e2292840bf29ed1cR58

Reply:
This otherwise errors on the `Pkg.add` call in the `catch` below.